src/main/java/org/kuali/kfs/module/ar/businessobject/TimeBasedBillingPeriod.java

```diff
@@ -83,6 +83,21 @@ protected Date determineStartDateByFrequency() {
         LOG.info("determineStartDateByFrequency: returning calculateNextDay(lastBilledDate) where lastBilledDate = " + lastBilledDate);
         return calculateNextDay(lastBilledDate);
     }
+    
+    /*
+     * CU Customization
```

Comment:
since the other places in this PR references CU-MOD instead of CU Customization, I think we may want to be consistent in these comments, to make it easier to find things